README.rst

```diff
@@ -50,7 +50,10 @@ Q003 Change outer quotes to avoid escaping inner quotes
 Configuration
 -------------
 
-By default, we expect single quotes (') and look for unwanted double quotes ("). To expect double quotes (") and find unwanted single quotes ('), use the CLI option:
+By default, we expect single quotes (') and look for unwanted double quotes ("), and
```

Comment:
I appreciate the effort here! I think this is a little more confusing though, since we're starting to talk about expectations in 2 different places.

It'd prob be better to split up each of the code samples with their relevant default and override settings (e.g. multiline, docstring).

Or simply add a default explainer to the existing comment lines

```
# We also support configuration for multiline quotes (default: """)

# We also support docstring quotes similarly (default: """)
```

What are your thoughts?

> For personal reference: https://github.com/zheller/flake8-quotes/blob/3.3.1/flake8_quotes/__init__.py#L137-L139

Reply:
So, we can go deeper here, but I think it's quite enough now just to write default behaviour, it will solve my and other developers misunderstanding.